demo/ThemeTransform.html

```diff
@@ -0,0 +1,109 @@
+      }
+      #demoMenu a {
+        display: block;
+        color: black;
+      }
+      #demoTools {
+        background: #ccc;
+        padding: 15px;
+        position: fixed;
+        left: 20px;
+        top: 20px;
+        z-index: 10000;
+        width: 230px;
+        border-radius: 8px;
+      }
+      .demoArticleTitle {
+        color: red;
+        font-size: 2.0em;
+      }
+    </style>
+ 
+    <link href="https://en.wikipedia.org/w/load.php?debug=false&lang=en&only=styles&skin=vector&modules=skins.minerva.base.reset|skins.minerva.content.styles|ext.math.styles|" rel="stylesheet" type="text/css"></link>
+
+    <script src=build/wikimedia-page-library-transform.js></script>
+    <link rel=import href=ControlPanel.html>
+  </head>
+
+  <body>
+    <div id='demoTools'>
+      <control-panel></control-panel>
+      <div id='demoMenu'></div>
+    </div>
+    <div class='content' id='content'></div>
+    
+    <script>
+        
+(function() {
+
+  // The json files below were saved using the mobileview parameters in the URL below.
+  // eslint-disable-next-line max-len
+  // https://en.wikipedia.org/w/api.php?action=mobileview&format=json&noheadings=true&page=Playwright&pilicense=any&prop=sections%7Ctext%7Clastmodified%7Clastmodifiedby%7Clanguagecount%7Cid%7Cprotection%7Ceditable%7Cdisplaytitle%7Cthumb%7Cdescription%7Cimage%7Crevision%7Cnamespace&sectionprop=toclevel%7Cline%7Canchor%7Clevel%7Cnumber%7Cfromtitle%7Cindex&sections=all&thumbwidth=640
+
+  const fileNames = [
+    'en.Barack_Obama.json',
+    'en.Audioslave.json',
+    'en.Chris_Cornell.json',
+    'en.Gantz.json',
+    'en.Mycenaean_Greek.json',
+    'en.Kill_\'Em_All.json',
+    'en.Manchester_United_F.C..json',
+    'en.John_McCain.json',
+    'en.Playwright.json',
+    'en.Toast.json'
+  ]
+
+  const urls = fileNames.map(filename => `/DemoArticles/${filename}`)
+
+  const demoMenu = document.getElementById('demoMenu')
+  fileNames.forEach(url => {
+    const titleAnchor = document.createElement('a')
+    titleAnchor.href = `#${url}`
+    titleAnchor.innerHTML = url
+    demoMenu.appendChild(titleAnchor)
+  })
+
+  // Layout the article section data consecutively on a single page.
+  // Comment out any files above you aren't interested in testing, then when finished double-check
+  // all of them.
+  Promise.all(urls.map(url => fetch(url).then(resp => resp.json())))
+    .then(jsons => jsons.map(json => json.mobileview.sections))
+    .then(sections => [].concat(...sections))
+    .then(sections =>
+      sections.map(section => `<div class='content_block' id='${section.id}'>
+                                <h2>${section.line}</h2>
+                                <br>${section.text}
+                              </div>`
+      )
+    )
+    .then(sections => sections.join(''))
+    .then(text => {
+      document.querySelector('#content').innerHTML = text
+
+      Array.from(document.querySelectorAll('.content_block[id="0"] h2:first-of-type'))
```

Comment:
Is Array.from() needed in the newer browsers we're targeting? Won't there only be a single result anyway (can we use `querySelector()` instead)?